pkgs/os-specific/darwin/apple-source-releases/diskdev_cmds/meson.build.in

```diff
@@ -8,7 +8,6 @@ project('diskdev_cmds', 'c', 'objc', version : '@version@')
 # Dependencies
 cc = meson.get_compiler('c')
 
-apfs = dependency('appleframeworks', modules : 'APFS')
```

Comment:
Why is the APFS framework being removed?

Reply:
It doesn’t seem to link any symbols, and apparently doesn’t even need anything from the headers:
https://github.com/NixOS/nixpkgs/blob/7994dc3c35ca1815902ec868b96036261725b865/pkgs/os-specific/darwin/apple-source-releases/diskdev_cmds/package.nix#L31-L33
As far as I can tell from the source, all the direct APFS API calls are guarded behind iOS conditionals. No point doing the private framework linking dance for programs that don’t use them. We could probably even just add the appropriate `#if`s to their `#include`s instead.